### PR TITLE
[merge] feat/ui-testing -> dev/frontend

### PR DESCRIPTION
**Pull request by HyungGeun-Cho**
#### Related Issue(s):
#73 
#### PR Description:
- Implemented UI testing for frontend
##### Changes Included:
- [x] Added new feature(s)
- [ ] Fixed identified bug(s)
- [ ] Updated relevant documentation
##### Screenshots (if UI changes were made):
Attach screenshots or GIFs of any visual changes. (Only for
frontend-related changes)
##### Notes for Reviewer:
테스트 시 text를 입력하기 위해 일부 TextInputEditText에 id를 추가했습니다.

---
#### Reviewer Checklist:
- [ ] Code is written in clean, maintainable, and idiomatic form.
- [ ] Automated test coverage is adequate.
- [ ] All existing tests pass.
- [ ] Manual testing has been performed to ensure the PR works as
expected.
- [ ] Code review comments have been addressed or clarified.

---
#### Additional Comments:
Add any other comments or information that might be useful for the
review process.

### CODE REVIEW

File: frontend/app/src/androidTest/java/com/project/spire/MainActivityUITest.kt [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+import org.junit.Before
+
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class MainActivityUITest {
+
+    @get:Rule
```
Comment (dayyyyyyyyyy) [inline frontend/app/src/androidTest/java/com/project/spire/MainActivityUITest.kt:42]: `@Rule`과 `@get:Rule`은 무슨 차이인가요?

**Review by dayyyyyyyyyy — APPROVED** [review]
수고하셨습니다! Conflict 해결하시고 머지하시면 될 것 같아요.

File: frontend/app/src/androidTest/java/com/project/spire/MainActivityUITest.kt (hunk 1 above)
Comment (HyungGeun-Cho) [reply frontend/app/src/androidTest/java/com/project/spire/MainActivityUITest.kt:42]: 코틀린 예시 코드에서 @get:Rule로 되어 있어서 해당 방식으로 작성하기는 했는데 추후 확인해보겠습니다..!

**Comment by tictactoeid** [comment]
수고하셨습니다!

**Pull request merged**